internal/controller/postgrescluster/controller.go

```diff
@@ -407,11 +406,24 @@ func (r *Reconciler) SetupWithManager(mgr manager.Manager) error {
 		}
 	}
 
+	var opts controller.Options
+
+	// TODO(cbandy): Move this to main with controller-runtime v0.9+
+	// - https://github.com/kubernetes-sigs/controller-runtime/commit/82fc2564cf
+	if s := os.Getenv("PGO_WORKERS"); s != "" {
+		if i, err := strconv.Atoi(s); err == nil && i > 0 {
+			opts.MaxConcurrentReconciles = i
+		} else {
+			mgr.GetLogger().Error(err, "PGO_WORKERS must be a positive number")
```

Comment:
Should we tell them that we're defaulting to 2?

Reply:
🤔 The `Starting workers` message seems like the right place for the 2. Maybe something about ignoring the variable here?

```
PGO_WORKERS must be a positive number; ignoring
PGO_WORKERS must be a positive number; using the default
Ignoring PGO_WORKERS: must be a positive number
Ignoring PGO_WORKERS: not a positive number
```